core/src/main/java/org/kiwix/kiwixmobile/core/history/HistoryActivity.java

```diff
@@ -60,6 +65,7 @@
   private final List<HistoryListItem> deleteList = new ArrayList<>();
   private static final String LIST_STATE_KEY = "recycler_list_state";
   public static final String USER_CLEARED_HISTORY = "user_cleared_history";
+  private AlertDialogShower alertDialogShower;
```

Comment:
This should be injected and it should be injected as the interface `DialogShower`